conduct.md

```diff
@@ -134,6 +150,8 @@ how long a decision will take to implement.
     * Deploy any kind of decision that is against codes of ethic.
     * Mock a team member's decision.
     * Refuse to do any assigned work in general.
+    * Share constructive criticism of others opinions before making a final decision as a group.
+    * Allow other group members to have an equal say in decisions.
```

Comment:
I think you meant to put this under the shall not section but it is showing up in the shall section. I may be seeing it wrong but double check this is in the right spot.